package.json

```diff
@@ -8,7 +8,7 @@
     "start": "NODE_ENV=production yarn run build && node ./dist/index.js",
     "lint": "eslint 'src/**/*.js' --fix",
     "start:dev": "NODE_ENV=development nodemon --exec babel-node src/index.js",
-    "test": "jest",
+    "test": "jest --passWithNoTests",
```

Comment:
I added `--passWithNoTests` because we didn't have any, but when I added a test file, `yarn` could not see any tests, interesting problem. 

I am still looking into it.